corehq/apps/export/models/new.py

```diff
@@ -211,6 +231,51 @@ def generate_instance_from_schema(schema, domain, app_id=None):
         return instance
 
 
+class ExportInstanceDefaults(object):
```

Comment:
instead of having a separate class for each level of the export instance just to have sensible defaults, i made a class just for handling defaults that Form and Case both subclass